cli/src/athenapdf.js

```diff
@@ -107,8 +113,9 @@ if (process.platform === "linux") {
     };
 }
 
+var pragmaHeader = athena.cache ? "" : "pragma: no-cache\n";
```

Comment:
I'm thinking, would it be better to stick to one HTTP header variable (i.e. `athena.httpHeader`), and simply push `pragma: no-cache` if `athena.cache` is enabled? Or would that be bad since `athena.httpHeader` is user-defined vs. `pragmaHeader` which is toggled?